pkg/didcomm/protocol/didexchange/didex_interop.go

```diff
@@ -0,0 +1,15 @@
+// +build ACAPyInterop
+
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package didexchange
+
+const (
+	// Interop: create "sov" docs when creating peer DIDs
```

Comment:
@Moopli Have you verified that AcaPy hasn't already fixed this?

Reply:
verified, they don't